Source/InitEB.cpp

```diff
@@ -1,6 +1,7 @@
 #include <memory>
 
 #include "EB.H"
+#include "EBLeastSquares.H"
```

Comment:
I don't think this warrants a new header. Can this go in the existing EB files?

Reply:
I could split the least-squares specific helper functions into Utilities.H or other headers, but I feel it will be hard to read. If this stays as a stand-alone header a developer has to just go through this one file. It will also not blow up the other files. So I prefer that it remain in a separate header.

Reply:
There are already a lot of EB utility functions in `EB.H` and `EB.cpp` so I think more would be fine. I feel there is an inconsistency with `pc_fill_bndry_grad_stencil_ls` being in `EBLeastSquares.H` and `pc_fill_bndry_grad_stencil_quadratic` in `EB.cpp` when they are so similar. Why not an `EBQuadratic.H` as well? (not that I'm suggesting that 🙂 )